purchase_deposit/wizard/purchase_make_invoice_advance.py

```diff
@@ -0,0 +1,196 @@
+# Copyright 2019 Elico Corp, Dominique K. <dominique.k@elico-corp.com.sg>
+# Copyright 2019 Ecosoft Co., Ltd., Kitti U. <kittiu@ecosoft.co.th>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import time
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+
+
+class PurchaseAdvancePaymentInv(models.TransientModel):
+    _name = 'purchase.advance.payment.inv'
+    _description = 'Purchase Advance Payment Invoice'
+
+    advance_payment_method = fields.Selection(
+        [('percentage', 'Down payment (percentage)'),
+         ('fixed', 'Deposit payment (fixed amount)')],
+        string='What do you want to invoice?',
+        default='percentage',
+        required=True,
+    )
+    purchase_deposit_product_id = fields.Many2one(
+        comodel_name='product.product',
+        string='Deposit Payment Product',
+        domain=[('type', '=', 'service')],
+    )
+    count = fields.Integer(
```

Comment:
Which is the usage of this field. I can figure out that could be used to create a deposit for multiple PO's, but I can't reproduce it.

Reply:
Yes, I wasn't needed. This part try to be similar to SO, but you are right, there is no such action in PO. So, I removed it.
Thank you @hveficent